clouddriver-cf/src/main/groovy/com/netflix/spinnaker/clouddriver/cf/model/CloudFoundryServerGroup.groovy

```diff
@@ -128,7 +131,7 @@ class CloudFoundryServerGroup implements ServerGroup, Serializable {
 
       @Override
       Map<String, Object> getBuildInfo() {
-        buildInfo
+        bi
```

Comment:
ah sorry - I was tricked by where the diff cut off and didn't notice the extra indentation